Octokit.Reactive/Clients/ObservablePackagesClient.cs

```diff
@@ -32,8 +30,12 @@ public ObservablePackagesClient(IGitHubClient client)
         public IObservable<Package> GetAll(string org, PackageType packageType, PackageVisibility? packageVisibility = null)
         {
             Ensure.ArgumentNotNullOrEmptyString(org, nameof(org));
+            Ensure.ArgumentNotNull(packageType, nameof(packageType));
```

Comment:
C# newbie question: when the method is declared with argument `PackageType packageType`, is it implicitly possible to pass null?

Reply:
No probs! No, an enum is a value type (like an int) so it can only be null if it’s specified with a ?

I had to look that one up and I’ve been using c# for around a decade 😨